pandas/core/indexes/category.py

```diff
@@ -133,7 +133,7 @@ def _create_from_codes(self, codes, categories=None, ordered=None,
         if name is None:
             name = self.name
         cat = Categorical.from_codes(codes, categories=categories,
-                                     ordered=self.ordered)
+                                     ordered=ordered)
```

Comment:
We’re there cases before where the wrong thing was passed? I.e. None != ordered != self.ordered?

Reply:
is this tested anywhere? I agree this should have broken things

Reply:
It appears that all instances of this private method `_create_from_codes`  in the codebase never passed an arg to `ordered` (i.e.  `ordered` always defaulted to `None` which always got reassigned to `self.ordered` here)

Reply:
```
(pandas-dev) matthewroeschke:pandas-mroeschke matthewroeschke$ grep -R --include="*.py" _create_from_codes .
./pandas/core/indexes/category.py:    def _create_from_codes(self, codes, categories=None, ordered=None,
./pandas/core/indexes/category.py:                new_target = self._create_from_codes(codes)
./pandas/core/indexes/category.py:        return self._create_from_codes(taken)
./pandas/core/indexes/category.py:        return self._create_from_codes(np.delete(self.codes, loc))
./pandas/core/indexes/category.py:        return self._create_from_codes(codes)
./pandas/core/indexes/category.py:        result = self._create_from_codes(codes, name=name)
./pandas/core/indexes/category.py:        # if name is None, _create_from_codes sets self.name
```